udata/harvest/actions.py

```diff
@@ -28,12 +28,16 @@ def list_backends():
     return backends.get_all().values()
 
 
-def list_sources(owner=None):
+def list_sources(owner=None, page=None, page_size=None):
     '''List all harvest sources'''
     sources = HarvestSource.objects.visible()
     if owner:
-        return list(sources.owned_by(owner))
-    return list(sources)
+        sources = sources.owned_by(owner)
+    if page_size:
+        page = max(page or 1, 1)
+        return sources.paginate(page, page_size)
```

Comment:
It is also used by cli which doesn't support pagination. Only the API is using pagination.